ada/language/ast.py

```diff
@@ -4152,7 +4152,9 @@ def is_definite_subtype():
     )
 
     discriminants_list = AbstractProperty(
-        type=BaseFormalParamDecl.entity.array
+        type=BaseFormalParamDecl.entity.array,
+        doc="Return the list of all discriminants of this type.",
```

Comment:
Can you extend this documentation? Specify what is returned when there are no discriminants and when discriminants are unknown.